### ISSUE TITLE
note about "being static" is not consistent with linked post

### ISSUE THREAD

**Issue by pnkfelix** [tokio-rs/website#518]
This text from the Spawning tutorial is trying to help correct a common misconception:

https://github.com/tokio-rs/website/blob/344f25dc9c28c84771504cc32e05f984b07e2e02/content/tokio/tutorial/spawning.md#L184-L187

but the terms this part of the tutorial uses are inconsistent with (and arguably contradict) the post that the tutorial links, which makes the note more confusing, not less.

Specifically, the info on the tutorial effectively claims that "being static" does not mean "lives forever".

But the linked document carefully distinguishes between two things: a type *with* a `'static` lifetime, and a type *bounded by* a `'static` lifetime. In the context of that doc, the former is the thing that is valid for the whole program ("lives forever"); the latter is the thing that might not.
 * (I'm not saying I personally agree with that choice of terminology; e.g. I think a reasonable person might think the type `&'a &'static T` is one "with" a static lifetime. But I digress...)

In any case, f I were asked where of the two terms ("with static lifetime" vs "bounded by static lifetime") sounds most like the term "this type is static", I would say "that sounds like a closer match to the former to me."

----

Okay, so what's my point?

My point is: I suggest the text of the tutorial try to avoid using the phrase "being static" or "making it static", et cetera. It should say "is `'static-bounded", to make it clear that it is talking about the `T: 'static` constraint.

(Or it could say such values must hold no references to temporary data.).

(If you are concerned that this is all too wordy, and that you need a term that is as convenient as "static values" here, then I might suggest you use the phrase "sendable values" instead.)

**Comment by Darksonn** [comment]
Thanks for the feedback! I'm definitely open to making the wording more precise, and I think this is an area that deserves being wordy! If you want, you can submit a PR, otherwise I'll try to get around to it at some point.

**Comment by Darksonn** [comment]
Note that my experience with this terminology in most of the Rust community is that "being `'static`" consistently refers to `T: 'static` among people who have used Rust a lot and are confident with these things. Still, it is certainly worth to expand on this in the guide.
